plsync/planetlab/model.py

```diff
@@ -30,6 +30,17 @@ def pl_interface(host_index, v4prefix):
     interface['ip']         = '%s.%d' % (net_prefix, mlab_offset)
     return interface
 
+def split_prefix(v4prefix):
```

Comment:
Could this be more concise with a regex?  The regex could even be used to 100% validate the IP address by bounding the characters and each octet.  Maybe not worth the effort... just an idea.

matches = re.match('^(\d+\.\d+\.\d+)\.(\d+)$', v4prefix)
if matches:
    return (matches.group(1), matches.group(2))